src/controllers/v1/entities.js

```diff
@@ -606,13 +614,148 @@ module.exports = class Entities extends Abstract {
 		})
 	}
 
+    /**
+     * @api {GET} v1/entities/subEntityList/663339bc0cb19f01c459853b?type=school&search=&page=1&limit=100
+     * Get sub entity list for the given entity. 
+     * @apiVersion 1.0.0
+     * @apiGroup Entities
+     * @apiHeader {String} X-authenticated-user-token Authenticity token
+     * @apiSampleRequest v1/entities/subEntityList/663339bc0cb19f01c459853b?type=school&search=&page=1&limit=100
+     * @apiUse successBody
+     * @apiUse errorBody
+     * @apiParamExample {json} Response:
+     * {
+    "status": 200,
+    "result": {
+        "data": [
+            {
+                "_id": "627a13928ce12806f5803f57",
+                "entityType": "school",
+                "externalId": "entity123",
+                "label": "undefined - entity123",
+                "value": "627a13928ce12806f5803f57"
+            }
+        ],
+        "count": 1
+    }
+    }
+
+    /**
+      * Get the immediate entities .
+      * @method
+      * @name subEntityList
+      * @param  {Request} req request body.
+      * @param {String} req.params._id - entityId
+      * @returns {JSON} Returns list of immediate entities
+     */
+
+    subEntityList(req) {
+        
+		return new Promise(async (resolve, reject) => {
+		    // Check if required parameters (_id or entities) are missing
+		  if( !(req.params._id || req.body.entities) ) {
+			return resolve({
+			  status :  HTTP_STATUS_CODE.bad_request.status,
+			  message : constants.apiResponses.ENTITY_ID_NOT_FOUND
+			})
+		  }
+	  
+			try {
+		    // Call 'entitiesHelper.subEntityList' to retrieve sub-entities based on the request parameters
+			  let entityDocuments = await entitiesHelper.subEntityList(
```

Comment:
@MallanagoudaB API is tested with pagination and search capability? If not test once

Reply:
Yes, both in search capability i applied as empty then i was getting some respose when i added number then it was not giving me response so search capability is tested .